### DISCUSSION TITLE
[UNKNOWN] e.response is undefined

### DISCUSSION THREAD

**Discussion by scouech** [directus/directus#4776] [Archive]
I have multiple directus on the same server.
One project is on port 8055, the other on port 8056. 

For the project running on 8055 everything works, for the project on 8056, despite having the right .env file with port 8056 when I access directus via my domain name it gives me an error after logging in. 

If I use the server IP address it works.. So the problem occurs only when using a domain name. 

Here is the error I'm getting:
<img width="580" alt="Screenshot 2021-03-31 at 1 23 57 PM" src="https://user-images.githubusercontent.com/26603052/113137065-5b1c6e80-9224-11eb-9bc7-62aa4d87e851.png">

```
{
  "fileName": "https://{{domain}}/admin/js/app.0038e4e5.js",
  "lineNumber": 1,
  "columnNumber": 451712,
  "message": "e.response is undefined"
}
```
Any ideas? 

Cheers

**Comment by rijkvanzanten** [comment]
> If I use the server IP address it works.. So the problem occurs only when using a domain name

This to me means that assuming something must be getting a little weird in the server setup for the reverse proxy to the domain 🤔 

What are you using for the server config to hook up the domains?

**Reply by scouech** [reply]
hey @rijkvanzanten !

I'm using nginx. 

Here is the config:

```
server {
    listen 80;
    server_name {{domain here}};

    location / {
        proxy_pass http://localhost:8056;
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
    }
}
```

The other directus project has the same config file, just the server_name and proxy_pass are different.

**Reply by rijkvanzanten** [reply]
Hmm that does look to me on first glance.

Are there any errors logged in the network tab? I'm assuming that one of the requests is failing because response doesn't exist, and I'm hopeful we can learn some more from the specific network error thrown 🙂

**Reply by scouech** [reply]
mmmm , network tab is now my favorite thing haha! 
This is what I'm getting (see below).. It seems that ublock (firefox extension) is blocking this.. It doesn't happen with the other directus instance.. very odd..
Do you know why? 
I'm stupid, I should have tried on another browser.. because everything was fine with the other project I just didn't think about doing that.. My bad! 

<img width="1132" alt="Screenshot 2021-03-31 at 4 14 11 PM" src="https://user-images.githubusercontent.com/26603052/113158505-23b9bc00-923c-11eb-9def-ceeca32d5a7f.png">

**Reply by rijkvanzanten** [reply]
Haha I'm pretty sure your adblocker is actually breaking Directus in this case, because of your domain `analytics.example.com` 😄 

I think the only thing you can really do is either disable your adblocker, or use a different subdomain than `analytics`

**Reply by scouech** [reply]
omg that's ridiculous hahahah! I'm sorry! I spent like hours trying to figure out when I could have checked the network tab lol. Thanks for your help!!! Well I guess I'm gonna change the subdomain haha!

**Reply by scouech** [reply]
Changed the subdomain and it works fine ;) stupid me! Thanks again

**Reply by rijkvanzanten** [reply]
Glad to hear!

> stupid me! 

Nahh, happens to every developer at least once 😄 

To me, it's in the list of:

* Cleared cache?
* No extensions?
* Tried turning it off and on again?

😂

**Reply by scouech** [reply]
Gonna write it on a post-it and stick it next to my screen haha!
